server/db/queries.js

```diff
@@ -0,0 +1,15 @@
+const knex = require('./db');
```

Comment:
As we're using an ORM I think having a separate queries file is a little redundant?

Reply:
I see your point, also the queries in this file are not used anywhere. Let me remove it to clean up the files.